lib/plugins/aws/provider/awsProvider.js

```diff
@@ -428,7 +428,13 @@ class AwsProvider {
         },
         provider: {
           properties: {
-            apiGateway: { type: 'object', properties: { websocketApiId: { type: 'string' } } },
+            apiGateway: {
+              type: 'object',
+              properties: {
+                websocketApiId: { type: 'string' },
+                shouldStartNameWithService: { type: 'boolean' },
```

Comment:
Let's set it as `const: true`. If we would allow also `false`, it'll mean that we should keep support for old way of things in v3.